src/dump.rs

```diff
@@ -58,6 +58,7 @@ impl RustDocCommand {
         &self,
         manifest_path: &std::path::Path,
         pkg_spec: Option<&str>,
+        all_features: bool,
```

Comment:
From what I see, only the placeholder project uses the `pkg_spec` argument, so the third argument is not needed (based on the second argument it is possible to deduce whether to use `--all-features`). But then the code would be less readable. What do you think @obi1kenobi ?

Reply:
I think being more explicit like this is better. I'd rather not debug this issue again, and I think if we try to be "clever" here and combine the meanings of the parameters like that, that's exactly what's likely to happen.

Reply:
Sure. So I guess everything's ready to be merged.

Reply:
Set to auto-merge when the tests finish, I merged the `main` branch into this one so it'll re-run CI one more time.

As soon as the PR with tests for this merges, we should make another release because this is an important bugfix that will enable cargo-semver-checks to work for crates that rely on features.